### ISSUE TITLE
Determine if activity/fragment is firstly created or re-created in @AfterViews

### ISSUE THREAD

**Issue by fikr4n** [androidannotations/androidannotations#1947]
*Description of the feature*

Sometimes we need to do something based on whether the activity/fragment is firstly created or re-created (e.g. configuration changes). Usually I check it in `onCreate` by examining whether the `Bundle` is null or not. Since in `@AfterViews` there is no `Bundle` argument, I have an idea to support such that goal. The code below is just my idea, maybe you have a better idea (or not to accept the idea).

*List and usages of the new annotation(s)*

No new annotation, only allowing `@AfterViews` annotated method to have a `boolean` argument.

```
    @AfterViews
    void initViews(boolean firstlyCreated) {
        ...
    }
```

*Examples of the generated code*

```
    ...

    private boolean isFirstlyCreated_;

    @Override
    public void onCreate(Bundle savedInstanceState) {
        ...
        isFirstlyCreated_ = savedInstanceState == null;
    }

    @Override
    public void onViewChanged(HasViews hasViews) {
        ...
        initViews(isFirstlyCreated_);
    }
```

**Comment by WonderCsabo** [comment]
I agree, this is a valid request. However, i would pass the `Bundle` instance, not just a `boolean`. @dodgex WDYT?

**Comment by dodgex** [comment]
While I see the goal - knowing if an Activity or Fragment is newly created or recreated - as a valid request. I'm not sure if `@AfterViews` is a good place? but as i don't know a better place i think it is okay.

I think I'd prefer to pass the `Bundle` too. but we have to ensure to `null` the reference after all `@AfterViews` methods have been called. (the sample is missing the change of the boolean)

**Comment by fikr4n** [comment]
Quoting @dodgex 

> I think I'd prefer to pass the Bundle too.

Today I found it's a great idea too. When using a library which needs to restore instance state by passing it `savedInstanceState` I think that I need the `Bundle` like you said, because I restore the library's component's views in `@AfterViews void initViews()`.
